app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
-<html class="no-js">
+<html>
   <head>
     <title>Leave and Travel Requests</title>
+    <style>html{visibility: hidden;opacity:0;}</style>
```

Comment:
Will adding the style here deal with the browser caching the page and the flash return

Reply:
I'm not sure I understand the problem you're asking about. I think this is more about the order in which the CSS is loaded. Does browser caching affect the order in which the CSS is loaded?